Capstone/API/APIManager.m

```diff
@@ -5,9 +5,19 @@
 //  Created by Abel Kelbessa on 7/5/22.
 //
 
-#import "APImanager.h"
+#import "APIManager.h"
 
-@implementation APImanager
+@implementation APIManager
+
+
++ (instancetype)shared {
+    static APIManager *sharedManager = nil;
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        sharedManager = [[self alloc] init];
+    });
+    return sharedManager;
+}
```

Comment:
We discussed this a little bit, and I *think* I see the potential use case — but I am wondering what you think the purpose of this will be 🤔